ProjectScripts/Modules/SPDI/module5_SPDI.py

```diff
@@ -12,23 +12,25 @@
 
 # Request parser to identify specific content-type requests
 parser = reqparse.RequestParser()
-parser.add_argument('content-type', type=str, help='Accepted\napplication/json')
+#parser.add_argument('content-type', type=str, help='Accepted\napplication/json')
 
 # Define a namespace for variant data
 va_space = api.namespace('SPDI', description='SPDI API Endpoints')
 
 
 @va_space.route(
-    "/hgvs/<string:hgvs>/<string:contextuals>")
+    "/hgvs/<string:hgvs>")
+#"/hgvs/<string:hgvs>/<string:contextuals>") removed contextuals as essential required input
 class VariantAnnotations(Resource):
 
     @api.doc(params={"hgvs": "HGVS format e.g NC_000017.10:g.48275363C>A",
                      "contextuals": "assembly"}, parser=parser)
 
+
     def get(self, hgvs, contextuals):
         """Retrieve varant data from the Leiden Open Variation Database (LOVD) using the VariantValidator API"""
```

Comment:
Replace Docstring with """Retrieve SPDI format for HGVS genomic variant from NCBI Variation Services SPDI API"